
from PyQt6 import QtCore, QtGui, QtWidgets
import resource_rc

class Ui_Padavan_l1_1(object):
    def setupUi(self, Padavan_l1_1):
        Padavan_l1_1.setObjectName("Padavan_l1_1")
        Padavan_l1_1.resize(915, 657)
        Padavan_l1_1.setMaximumSize(QtCore.QSize(915, 657))
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap(":/recurce/icon.png"), QtGui.QIcon.Mode.Normal, QtGui.QIcon.State.Off)
        Padavan_l1_1.setWindowIcon(icon)
        self.centralwidget = QtWidgets.QWidget(Padavan_l1_1)
        self.centralwidget.setMinimumSize(QtCore.QSize(915, 657))
        self.centralwidget.setStyleSheet("background-color: rgb(255, 255, 255);")
        self.centralwidget.setObjectName("centralwidget")
        self.gridLayout = QtWidgets.QGridLayout(self.centralwidget)
        self.gridLayout.setContentsMargins(15, 11, -1, 11)
        self.gridLayout.setObjectName("gridLayout")
        self.frame = QtWidgets.QFrame(self.centralwidget)
        self.frame.setMinimumSize(QtCore.QSize(445, 0))
        self.frame.setMaximumSize(QtCore.QSize(445, 16777215))
        self.frame.setFrameShape(QtWidgets.QFrame.Shape.StyledPanel)
        self.frame.setFrameShadow(QtWidgets.QFrame.Shadow.Raised)
        self.frame.setObjectName("frame")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.frame)
        self.verticalLayout.setObjectName("verticalLayout")
        self.label_4 = QtWidgets.QLabel(self.frame)
        self.label_4.setMaximumSize(QtCore.QSize(16777215, 45))
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(21)
        font.setBold(True)
        font.setWeight(75)
        self.label_4.setFont(font)
        self.label_4.setObjectName("label_4")
        self.verticalLayout.addWidget(self.label_4)
        self.label = QtWidgets.QLabel(self.frame)
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(19)
        self.label.setFont(font)
        self.label.setStyleSheet("color: rgb(95, 136, 240);")
        self.label.setObjectName("label")
        self.verticalLayout.addWidget(self.label)
        self.gridLayout.addWidget(self.frame, 1, 2, 1, 1)
        self.line = QtWidgets.QFrame(self.centralwidget)
        self.line.setLayoutDirection(QtCore.Qt.LayoutDirection.LeftToRight)
        self.line.setFrameShape(QtWidgets.QFrame.Shape.VLine)
        self.line.setFrameShadow(QtWidgets.QFrame.Shadow.Sunken)
        self.line.setObjectName("line")
        self.gridLayout.addWidget(self.line, 1, 1, 1, 1)
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setMinimumSize(QtCore.QSize(421, 483))
        self.label_2.setMaximumSize(QtCore.QSize(421, 483))
        self.label_2.setObjectName("label_2")
        self.gridLayout.addWidget(self.label_2, 1, 0, 1, 1)
        self.frame_2 = QtWidgets.QFrame(self.centralwidget)
        self.frame_2.setMaximumSize(QtCore.QSize(16777215, 50))
        self.frame_2.setFrameShape(QtWidgets.QFrame.Shape.StyledPanel)
        self.frame_2.setFrameShadow(QtWidgets.QFrame.Shadow.Raised)
        self.frame_2.setObjectName("frame_2")
        self.horizontalLayout = QtWidgets.QHBoxLayout(self.frame_2)
        self.horizontalLayout.setContentsMargins(-1, 0, -1, 0)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.pushButton_4 = QtWidgets.QPushButton(self.frame_2)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.pushButton_4.sizePolicy().hasHeightForWidth())
        self.pushButton_4.setSizePolicy(sizePolicy)
        self.pushButton_4.setMinimumSize(QtCore.QSize(0, 40))
        self.pushButton_4.setMaximumSize(QtCore.QSize(16777215, 40))
        self.pushButton_4.setStyleSheet("image:url(:/recurce/map.png);\n"
"border: none;")
        self.pushButton_4.setText("")
        self.pushButton_4.setObjectName("pushButton_4")
        self.horizontalLayout.addWidget(self.pushButton_4)
        self.pushButton = QtWidgets.QPushButton(self.frame_2)
        self.pushButton.setMinimumSize(QtCore.QSize(0, 40))
        self.pushButton.setMaximumSize(QtCore.QSize(16777215, 40))
        self.pushButton.setStyleSheet("image: url(:/recurce/book.png);\n"
"border: none;")
        self.pushButton.setText("")
        self.pushButton.setObjectName("pushButton")
        self.horizontalLayout.addWidget(self.pushButton)
        self.pushButton_2 = QtWidgets.QPushButton(self.frame_2)
        self.pushButton_2.setMinimumSize(QtCore.QSize(0, 30))
        self.pushButton_2.setMaximumSize(QtCore.QSize(16777215, 30))
        self.pushButton_2.setStyleSheet("image: url(:/recurce/next_btn.png);\n"
"border: none;")
        self.pushButton_2.setText("")
        self.pushButton_2.setObjectName("pushButton_2")
        self.horizontalLayout.addWidget(self.pushButton_2)
        self.pushButton_3 = QtWidgets.QPushButton(self.frame_2)
        self.pushButton_3.setMinimumSize(QtCore.QSize(0, 40))
        self.pushButton_3.setMaximumSize(QtCore.QSize(16777215, 40))
        self.pushButton_3.setStyleSheet("image: url(:/recurce/home.png);\n"
"border: none;")
        self.pushButton_3.setText("")
        self.pushButton_3.setObjectName("pushButton_3")
        self.horizontalLayout.addWidget(self.pushButton_3)
        self.gridLayout.addWidget(self.frame_2, 2, 0, 1, 3)
        self.frame_3 = QtWidgets.QFrame(self.centralwidget)
        self.frame_3.setMinimumSize(QtCore.QSize(0, 50))
        self.frame_3.setMaximumSize(QtCore.QSize(16777215, 60))
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(23)
        font.setBold(True)
        font.setWeight(75)
        self.frame_3.setFont(font)
        self.frame_3.setFrameShape(QtWidgets.QFrame.Shape.StyledPanel)
        self.frame_3.setFrameShadow(QtWidgets.QFrame.Shadow.Raised)
        self.frame_3.setObjectName("frame_3")
        self.verticalLayout_2 = QtWidgets.QVBoxLayout(self.frame_3)
        self.verticalLayout_2.setContentsMargins(0, 0, 0, 15)
        self.verticalLayout_2.setObjectName("verticalLayout_2")
        self.label_3 = QtWidgets.QLabel(self.frame_3)
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(23)
        font.setBold(True)
        font.setWeight(75)
        self.label_3.setFont(font)
        self.label_3.setObjectName("label_3")
        self.verticalLayout_2.addWidget(self.label_3)
        self.gridLayout.addWidget(self.frame_3, 0, 0, 1, 3)
        Padavan_l1_1.setCentralWidget(self.centralwidget)

        self.label_4.setStyleSheet("font-weight: bold")
        self.label_3.setStyleSheet("font-weight: bold")

        self.retranslateUi(Padavan_l1_1)
        QtCore.QMetaObject.connectSlotsByName(Padavan_l1_1)

    def retranslateUi(self, Padavan_l1_1):
        _translate = QtCore.QCoreApplication.translate
        Padavan_l1_1.setWindowTitle(_translate("Padavan_l1_1", "Level 1"))
        self.label_4.setText(_translate("Padavan_l1_1", "<html><head/><body><p align=\"center\">Тест 1. Вывод данных</p></body></html>"))
        self.label.setText(_translate("Padavan_l1_1", "<html><head/><body><p align=\"center\">Ожидание появления</p><p align=\"center\">заданий</p></body></html>"))
        self.label_2.setText(_translate("Padavan_l1_1", "<html><head/><body><p><img src=\":/recurce/Padavan_l1_1_image.png\"/></p></body></html>"))
        self.label_3.setText(_translate("Padavan_l1_1", "<html><head/><body><p align=\"center\"><span style=\" font-size:22pt;\">Подземелье</span></p></body></html>"))

